cmd/main.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus-community/json_exporter/config"
 	"github.com/prometheus-community/json_exporter/exporter"
+	"github.com/prometheus-community/json_exporter/https"
```

Comment:
This should be `github.com/prometheus/exporter-toolkit`, so that the Go module is included, rather than copy-n-paste of the code.